server/java/src/main/java/com/stripe/sample/Server.java

```diff
@@ -54,7 +54,7 @@ public static void main(String[] args) {
         Stripe.apiKey = dotenv.get("STRIPE_SECRET_KEY");
 
         staticFiles.externalLocation(
-                Paths.get(Paths.get("").toAbsolutePath().toString(), dotenv.get("STATIC_DIR")).normalize().toString());
+                Paths.get(Paths.get("").toAbsolutePath().toString(), dotenv.get("STATIC_DIR"), "html").normalize().toString());
```

Comment:
Instead of putting "html" here, we should use the STATIC_DIR environment variable to maintain this. When installing a Sample with the Stripe CLI, it'll squash the directory structure for the client side and there will not be an `html` directory, just `client`.  (ditto for all the other static file setup changes in this PR)